### PR TITLE
Style improvements to prevent line hop on small screens

### PR DESCRIPTION
**Pull request by Bayernatoor**
This should prevent line hops on very small mobile screens. Further resolves: https://github.com/OpenSats/website/issues/324 

1. add custom breakpoint to avoid line hop on small screens
2. some style updates

@Arvin21M Mind giving this a test on mobile when you get a chance. Should fix the previous issue.

### CODE REVIEW

**Comment by dergigi** [comment]
Code looks good to me and I just gave it a spin using the `iPhone SE` simulator in Chrome, but I'll defer to @Arvin21M to do the final testing before we merge this.

**Comment by Arvin21M** [comment]
@Bayernatoor and @dergigi Looks great on build preview! Good to go 🚀 

Thank you 🙏

**Review by Arvin21M — APPROVED** [review]

**Pull request merged**
